bin/generate-token.py

```diff
@@ -0,0 +1,8 @@
+import mongoengine
+
+from tcidatabase.models import BaseToken, SurveyToken, Client, User
+
+mongoengine.connect('', alias='default')
```

Comment:
You need to connect the the database in order to have the models to work.

See [a connection example here](https://github.com/anthropedia/tci-online/blob/master/core/__init__.py#L15)